Hindawi Publishers.js

```diff
@@ -49,6 +49,11 @@ function detectWeb(doc,url) {
 	return false;
 }
 
+function decodeEntities(doc, encodedString) {
```

Comment:
https://github.com/zotero/zotero/blob/4.0/chrome/content/zotero/xpcom/utilities.js#L451 should do what you want